src/jg_ast_optimize.ml

```diff
@@ -23,23 +23,23 @@ let dead_code_elimination stmts =
   let rec maybe_set = function
     | SetExpr set -> List.iter maybe_set set
     | IdentExpr id -> set_local id
-    | KeywordExpr (id, _) -> maybe_set id
+    (* | KeywordExpr (id, _) -> maybe_set id *)
     | _ -> () in
   let statement self = function
     | SetStatement (id, _) as s ->
       maybe_set id ;
       default_mapper.statement self s
     | ForStatement (id, _, _) as s ->
```

Comment:
Now `Tset` becomes just a list internally, so `id` is list, right?

Then please use `ids` instead of `id` like `ast_mapper`.

Reply:
Well spotted.